manifests/03-rbac-role-cluster.yaml

```diff
@@ -68,3 +68,16 @@ rules:
   - create
   - update
   - delete
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: console
```

Comment:
```suggestion
  name: console-customresourcedefinitions-reader
```

Reply:
Sam preferred to keep it `console` so that we could add additional permissions later. You can see his comment above.